### PR TITLE
Creating file and image fields for Azure storage

### PR DESCRIPTION
**Pull request by aaronpowell**
Pretty much a copy of the S3 one, but for Azure.

Haven't been able to test locally as the example server just never starts.

### CODE REVIEW

File: packages/fields-azure/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# @k6-contrib/fields-azure
+
+## 0.1.0
```
Comment (gautamsi) [inline packages/fields-azure/CHANGELOG.md:3]: nothing should be in changelog except the header for package name

**Comment by gautamsi** [comment]
I have not been able to run these tests either :(

File: packages/fields-azure/CHANGELOG.md (hunk 1 above)
Comment (aaronpowell) [reply packages/fields-azure/CHANGELOG.md]: My bad, made it empty now

**Comment by aaronpowell** [comment]
I've tried running the example site as well, but it just hangs trying to hit http://localhost:3000 and I can't figure out why

**Comment by aaronpowell** [comment]
Example works, fixed a few bugs I found in the image field, now that I've seen it in action 🤣

**Comment by gautamsi** [comment]
Great, I will rest and merge over weekend. 

Are you able to create a PR for fixes you have for s3 field? or they are included here?

**Comment by aaronpowell** [comment]
There shouldn't be any changes to the s3 field, or at least, I won't have done any

**Pull request merged**